Tests/fixtures/smartphone-2.yml

```diff
@@ -8650,24 +8650,6 @@
     model: Studio X
   os_family: Android
   browser_family: Chrome
--
```

Comment:
the test dublicate
https://github.com/matomo-org/device-detector/blob/master/Tests/fixtures/smartphone-17.yml#L8349